openerp/openupgrade/deferred_80.py

```diff
@@ -187,7 +187,7 @@ def migrate_stock_move_warehouse(cr):
     cr.execute(
         "SELECT count(*) FROM ir_module_module WHERE name='stock' "
         "AND state='installed'")
-    if not cr.fetchone():  # No stock
+    if not cr.fetchone()[0]:  # No stock
```

Comment:
this solve the error i've already tested it"SELECT * FROM ir_module_module WHERE name='stock' "
         "AND state='installed'")